### PR TITLE
Audition page done

### PR DESCRIPTION
**Pull request by TheHiddenLayer**

### CODE REVIEW

**Review by jayrc7 — CHANGES_REQUESTED** [review]
The page looks really good! For the mobile design of it, the font size of the audition page should be a lot smaller, and I'm thinking it should also be at the top of the screen. Also, just show the second picture for the mobile version, no need to show both. Great job. 

image

File: src/components/AuditionEligibility.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                </ul>
+                </div>
+
+                <div>
+                <h6>Dynamos Junior Team eligibility:</h6>
+                <ul>
+                    <li>8-10 years old</li>
+                    <li>One year of dance experience.</li>
+                    <li>3 hours minimum technique at one or more dance studios: one hour of jazz, 
+                        one hour of ballet, and one hour of another genre of dance.em</li>
```
Comment (jayrc7) [inline src/components/AuditionEligibility.js]: Is there supposed to be .em at the end?

**Review by jayrc7 — CHANGES_REQUESTED** [review]

**Review by jayrc7 — CHANGES_REQUESTED** [review]
This looks really good! There's been a tiny change to that page, Danielle wants the pictures to switch places. After making that change the pr is good to go.

**Comment by TheHiddenLayer** [comment]
> This looks really good! There's been a tiny change to that page, Danielle wants the pictures to switch places. After making that change the pr is good to go.


### Switched the pictures
> see below
Screen Shot 2020-05-18 at 11 27 08 AM

**Review by jayrc7 — APPROVED** [review]
nice

**Review by Anshul-Birla — APPROVED** [review]
Looks good bruv

**Pull request merged**